lib/checkfunctions.cpp

```diff
@@ -188,7 +188,7 @@ void CheckFunctions::checkMathFunctions()
         for (const Token* tok = scope->classStart->next(); tok != scope->classEnd; tok = tok->next()) {
             if (tok->varId())
                 continue;
-            if (printWarnings) {
+            if (printWarnings && Token::Match(tok, "%name% ( %any% [,)]")) {
```

Comment:
This is true, but your pattern enforces that the first argument consists of a single token, which is not required for the fmod|fmodf|fmodl-branch.